ssh-pkcs11.c

```diff
@@ -77,6 +77,13 @@ struct pkcs11_key {
 	int			keyid_len;
 };
 
+struct pkcs11_keyinfo {
```

Comment:
Please move the windows specific code to #ifdef WINDOWS block. Please add comments on why there is a deviation.
Please take this as global comment for all the common files between Unix and Windows.